### ISSUE TITLE
DefaultHandler instead of DiscardHandler

### ISSUE THREAD

**Issue by samribeiro** [netrack/openflow#110]
Small nit but it seems intuitive to me that all references to DiscardHandler should instead be named DefaultHandler.

```
func DiscardHandler(rw ResponseWriter, r *Request) {}

var DefaultHandler = DiscardHandler

s/DiscardHandler/DefaultHandler
```

we can then override an externally visible DefaultHandler which makes more sense than overriding DiscardHandler.

**Comment by ybubnov** [comment]
@samribeiro, thank you for the created issue! Totally agree with your proposition, do you mind to create a patch for that?

**Comment by ybubnov** [comment]
Issue is closed by #111
